cocos/platform/ios/CCApplication-ios.mm

```diff
@@ -177,13 +191,17 @@ -(void) doCaller: (id) sender
 
 Application* Application::_instance = nullptr;
 
-Application::Application(const std::string& name)
+Application::Application(const std::string& name, int width, int height)
 {
     Application::_instance = this;
     _scheduler = new Scheduler();
 
     createView(name);
     
+    // Shoule create _renderTexture here, then we don't have to consider how to
+    // revert the GL states. It is difficult to revert the states.
+    _renderTexture = new RenderTexture(width, height, 2);
```

Comment:
Perhaps, `RenderTextureScreen` is a better name since current `RenderTexture` isn't a command module.
It's only for devicePixelRatio functionality.